packages/runtime/lib/scheduler.js

```diff
@@ -0,0 +1,109 @@
+const { CronJob, validateCronExpression } = require('cron')
```

Comment:
```suggestion
'undici'

const { CronJob, validateCronExpression } = require('cron')
```

Reply:
@mcollina Never saw that: what does it do?

Reply:
it's an old school module that implement in process "unix cron" support.

Reply:
Ehm... no, I was referring to the `undici` statement

Reply:
I think @mcollina wanted to write `use strict` :D

Reply:
Lol. It makes way more sense now. 🤣